cmd/mt-gateway/api.go

```diff
@@ -2,6 +2,7 @@ package main
 
 import (
 	"fmt"
+	"github.com/grafana/metrictank/cmd/mt-gateway/ingest"
```

Comment:
that's just a convention: we usually sort the imports so that there are first all the standard library imports, then an empty line, then all the imports of other pkgs. (f.e. https://github.com/grafana/metrictank/blob/master/idx/memory/memory.go#L3)

Reply:
186fab2a2c2d42fa4de174c437a2f11361fb69c2